addons/api/mirage/generated/factories/worker.js

```diff
@@ -13,6 +13,7 @@ import { faker } from '@faker-js/faker';
 export default Factory.extend({
   name: () => faker.word.words(),
   description: () => faker.word.words(),
+  type: 'pki',
```

Comment:
Non-blocking: For this type value should we import it from the model instead?

Reply:
Great point. I will update that.